src/player.cpp

```diff
@@ -3636,7 +3636,7 @@ int player::talk_skill() const
     /** @EFFECT_SPEECH increases talking skill */
     int ret = get_int() + get_per() + get_skill_level( skill_id( "speech" ) ) * 3;
     if (has_trait( trait_SAPIOVORE )) {
-        ret -= 20; // Friendly convo with your prey? unlikely
+        ret -= 20; // Friendly convolution with your prey? unlikely
```

Comment:
I read the whle diff so I could fix this one mistake, success!

Reply:
:1st_place_medal: @ZhilkinSerg
:2nd_place_medal: @kevingranade 
:3rd_place_medal: OA